src/main/java/com/softserveinc/speakukrainian/pageobjects/components/addCenterVisitor/AddLocationModal.java

```diff
@@ -75,11 +76,15 @@ public String getCityText() {
     }
 
     public AddLocationModal clickCityDropDown(int count) {
-        getCity().click();
```

Comment:
The method was clicking on the text label instead of the drop-down list. Fixed, and also added an additional getter for the dropdown list